lib/components/MenuItemDivider/MenuItemDivider.tsx

```diff
@@ -10,7 +10,7 @@ export const MenuItemDivider = () => {
   );
 
   return (
-    <Box padding="xxsmall">
+    <Box padding={menuPadding}>
```

Comment:
This does not seem to have worked as expected, the [preview site](https://braid-design-system--a54f00fdb765712084723ff2b15806d69ad594e7.surge.sh/components/MenuItemDivider) has dropped padding on the divider 🤔

Reply:
Also wondering whether we should be using `role="separator"` on here so when i user is navigating through using the reading keys the separator is announced to them.